RELEASE_NOTES.md

```diff
@@ -30,6 +30,19 @@ Please see [https://docs.chef.io/release_notes.html](https://docs.chef.io/releas
 
 * Support for Solaris releases before 10u11 has been removed
 * Upgraded Ohai to 8.20 with new / enhanced plugins. See the [ohai changelog](https://github.com/chef-cookbooks/ohai/blob/master/CHANGELOG.md)
+* Added cab_package resource and provider which supports the installation of CAB/cabinet files. Using this resource you can install and remove cab windows packages. Example:
+
+  ```ruby
+  cab_package 'Windows6.1-KB958488-x64' do
```

Comment:
you might change it in the example then to imply that, like
`cab_package 'Install .NET 3.5 sp1 via KB958488' do`